src/GraphQL/Conversion/ValueConverter.cs

```diff
@@ -49,9 +49,9 @@ static ValueConverter()
 
                 return Convert.ToBoolean(value, NumberFormatInfo.InvariantInfo).Boxed();
             });
-            Register<string, Guid>(value => Guid.Parse(value));
+            Register<string, Guid>(Guid.Parse);
```

Comment:
I would not want to change this signature to keep the overall style.

Reply:
Agreed